coalaip/plugin.py

```diff
@@ -34,6 +34,32 @@ def generate_user(self, *args, **kwargs):
         """
 
     @abstractmethod
+    def get_history(self, persist_id):
+        """Get the ownership history of an entity on the persistence
+        layer.
+
+        Args:
+            persist_id (str): Id of the entity on the persistence layer
+
+        Returns:
+            list of dict: The ownership history of the entity. Each dict
+            is of the form::
+
+                {
+                    'user': A representation of a user as specified by the
+                            persistence layer (may omit secret details, e.g. private keys),
```

Comment:
Something I noticed while adding this: right now, we only expect COALA IP entities to only have one owner at a time. We could (theoretically) add support for multiple owners, but do we right now?